app/views/converts/new.slim

```diff
@@ -0,0 +1,18 @@
+- title 'convert'
+- content_for(:header) do
+  h2 convert splits to other cool formats
+article
+  form#convert method="post" action="" enctype="multipart/form-data"
+    .form-group.col-sm-4
+      input type="file" name="file"
+    .form-group.col-sm-2
+      select name="format"
+        - (Run.programs - [Llanfair]).each do |prgrm|
+          option value="#{prgrm.to_sym}"
+            = prgrm.to_s
+        option value="json" JSON
+    .form-group.col-sm-2
+      input type="checkbox" name="historic" id="historic"
+      label for="historic" Include History?
```

Comment:
This checkbox looks squished up against the `I`; can we add a space here

![screenshot 2016-01-18 18 56 53](https://cloud.githubusercontent.com/assets/438911/12408438/4198c15e-be15-11e5-95a3-771e5ba683b8.png)